### PR TITLE
Feature/engine tweaks

### PR DESCRIPTION
**Pull request by Windvis**
The needed change to import styles and some small improvements.

### CODE REVIEW

**Comment by Windvis** [comment]
At the moment the app needs add the needed Ember Data models and Serializer + data table mixin. It seems weird to do, since the app doesn't need those themselves. The problem is that the engine can't provide that, since it's isolated.

I think a complementary addon which contains the models + serializer might be a good future improvement. That way every app can simply install that addon and extend the serializer.

**Review by aatauil — APPROVED** [review]
Thanks for the changes, everything looks a lot cleaner :1st_place_medal:

**Comment by aatauil** [comment]
Dont have access to ci in dockercloud so cant really see what is failing, @cecemel would you be able to see what checks fail?

File: package.json [hunk 1]
```diff
@@ -47,8 +47,11 @@
     "ember-cli-sri": "^2.1.1",
     "ember-cli-uglify": "^3.0.0",
     "ember-data": "~3.17.0",
+    "ember-data-table": "^2.0.0",
+    "ember-engines": "^0.8.14",
     "ember-export-application-global": "^2.0.1",
     "ember-fetch": "^7.0.1",
+    "ember-jobs-dashboard": "file:.yalc/ember-jobs-dashboard",
```
Comment (Windvis) [inline package.json:54]: Woops, that shouldn't have been checked in :grimacing:

**Pull request closed without merging**